### ISSUE TITLE
Eternal Darkness -Uuvana 4

### ISSUE THREAD

**Issue by Fulkrem** [Uuvana-Studios/longvinter-windows-client#100]
I've logged out three times now and I end up in darkness everytime I log back in. This is what it looks like, I am inside my house luckily as it was the last place I logged out at. 
![Loggingin](https://user-images.githubusercontent.com/100523347/156074583-68917e98-8a86-4acd-a6ff-f2a50fd62647.png)

**Comment by Fulkrem** [comment]
Logging out five times now and completely shutting the game off and relogging back in. I continue to be met with this. Again for detail I was sitting inside my house before I logged off last night. So if there was anything that got rid of houses or made em poof? This could be part of the problem perhaps. 

However, before it changed to this. The first time I had logged in, showed my furniture for a brief moment before returning to the above.
